src/App.css

```diff
@@ -1,39 +1,53 @@
-.App {
-  text-align: center;
+body {
+  width: 100%;
```

Comment:
- [ ] The `Calculator.js` component should match the following given design 👇🏻 Kindly make sure you use the same color as on the image shown bellow 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/183694423-dd4bd117-830c-4b8e-a90d-6cc1bc7fd7b0.png)

Reply:
## Thank you @BenMukebo for reviewing my project
I have learned how to convert functions to classes in React Js
I have also installed the **Color Picker Eyedropper extension** to my browser to use the exact given color in  my project.

**Thank you and Happy reviewing! :joy_cat:**